talent/hiring_process.md

```diff
@@ -0,0 +1,63 @@
+# Hiring Process Explained
+
+This document explains the internal hiring process at source{d} and the responsibilities for each person related to it.
+
+
+### Job Description
+
+- Job Description is written by the lead engineers with Anna's support. 
+- Then it should be reviewed by Maximo and Eiso via a PR on Minutes.
+- Anna is posting the job description in source{d} website.
+
+### Sourcing
+
+- Lead engineers are responsible to find sources of people to be contacted. 
+- At any given moment Anna is able to drop in and work on finding more candidates.
+- Sourcing channels should continuously be defined and refreshed after communication between Anna and the leads. 
+- In case there is lack of candidates, Anna is responsible to contact Francesc in order to gain people through community actions. 
+
+### First Contact
+
+- Any developer should be contacted by the lead engineers or under their name at least. In case Anna is contacting them, she should use their email address. It's a motivating factor for the candidates to be contacted by the lead engineers. 
+
+### Screening
+
+- If the candidate is an applicant, they should be accepted/rejected by the lead engineers during the screening phase. 
+
+- If the candidate is approached by us from a specific source (known from meetups, having worked together in the past,  etc.), they should move forward to the next step as it means that the leads had already shown interest. 
+
+### First Skype Interview
+
+- A quick Skype call should be made with every candidate who is interested in the roles. It is a general duscussion to know more about the background and experience of the candidate. 
+
+- Lead engineers are responsible to make the calls as it's a motivating factor for the candidates and also the leads will know from early on if they would like to work with them or not. 
+
+### Coding Challenges 
+
+- Every candidate who is passing the first Skype interview, should complete the coding challege. 
+
+- In case of Data Science, there is an additional ML task necessary to be solved.
+
+- Coding Challeges are being sent to the candidates by Anna and the results are being reviewed by the lead engineers. 
+
+### On-site Interview
+
+- This is the last step in which it's being decided if the candidate will receive an offer or not. It might last a full day or being split in two days, regarding the availability of the candidate. 
+
```

Comment:
Since this is going to be public. Do you think it would be a good idea to mention the three different interviews that will take place? Or is this something to be avoided?
For example
- **Q&A** to know each other,
- **Design interview** to see how the candidate would design a solution to a given technical problem working with a couple of our engineers
-  **Logical thinking** to see how the candidate solves an abstract problem in a logical way.